### DISCUSSION TITLE
Channel/Frequency Switch with one button

### DISCUSSION THREAD

**Discussion by Highforest2023** [Tunas1337/UV-K5-Modded-Firmwares#53] [Ideas]
First thanks for your great job ...

What I missing in the K5 is the possibilitiy to have a look which frequency is "behind" a labeled Channel.
Up to now I have to go to the menu, change from "name" to "frequency"; have a look and have to switch back again in the menu.

"One button" (like F 5 - in Europe no one need the NOAA or one of the programmable buttons) to have a short look and back.

What do you think, is it possible?

Bert

**Comment by Highforest2023** [comment]
Sorry, I found the solution by myself :-)

F *  (F SCAN); here you can see directly the frequency of the channel

**Reply by Tunas1337** [reply]
I actually tried to turn Fn+5 into a shortcut to toggle the "frequency/name" setting. I didn't get much success, but future attempts to re-engineer the firmware should prove more fruitful.

**Comment by lohtse** [comment]
Yeah 

> First thanks for your great job ...
> 
> What I missing in the K5 is the possibilitiy to have a look which frequency is "behind" a labeled Channel. Up to now I have to go to the menu, change from "name" to "frequency"; have a look and have to switch back again in the menu.
> 
> "One button" (like F 5 - in Europe no one need the NOAA or one of the programmable buttons) to have a short look and back.
> 
> What do you think, is it possible?
> 
> Bert

Yes it is possible and I see you found the solution... I only know  about it due ot the Telegram group where it is mentioned there.

**Comment by Highforest2023** [comment]
Thanks to youF * is ok for me--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 10.08.23, 13:30 schrieb Andrej ***@***.***>:

   
  I actually tried to turn Fn+5 into a shortcut to toggle the "frequency/name" setting. I didn't get much success, but future attempts to re-engineer the firmware should prove more fruitful.

**Comment by Highforest2023** [comment]
Hello Andrej, for me, the biggest disadvantage are the functions you need to use the F-Button first.Could it be possible to "switch" that the functions like A/B, VFO are directly useable and you need the F-Button to use the numbers.My Notebook has this function and I love it.This will solve the Scanfunction too.All the bestBerthold --Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 10.08.23, 13:30 schrieb Andrej ***@***.***>:

   
  I actually tried to turn Fn+5 into a shortcut to toggle the "frequency/name" setting. I didn't get much success, but future attempts to re-engineer the firmware should prove more fruitful.
